xarray/core/groupby.py

```diff
@@ -32,7 +32,6 @@
     filter_indexes_from_coords,
     safe_cast_to_index,
 )
-from .ops import IncludeCumMethods
```

Comment:
Can we delete `IncludeCumMethods` from `ops.py`?

Reply:
```
 ________________________ TestVariable.test_reduce_funcs ________________________
[gw2] linux -- Python 3.8.13 /home/runner/micromamba/envs/xarray-tests/bin/python

self = <xarray.tests.test_variable.TestVariable object at 0x7ff452c11550>

    def test_reduce_funcs(self):
        v = Variable("x", np.array([1, np.nan, 2, 3]))
        assert_identical(v.mean(), Variable([], 2))
        assert_identical(v.mean(skipna=True), Variable([], 2))
        assert_identical(v.mean(skipna=False), Variable([], np.nan))
        assert_identical(np.mean(v), Variable([], 2))
    
        assert_identical(v.prod(), Variable([], 6))
>       assert_identical(v.cumsum(axis=0), Variable("x", np.array([1, 1, 3, 6])))
E       AttributeError: 'Variable' object has no attribute 'cumsum'

/home/runner/work/xarray/xarray/xarray/tests/test_variable.py:1869: AttributeError
```
Does variable also need this?

Reply:
I think I'll undo this. This can be a goal for a future PR.

Reply:
Oh nice find. Thanks